assets/chat/js/menus.js

```diff
@@ -287,10 +287,13 @@ class ChatUserMenu extends ChatMenu {
     }
 
     addElement(username, sort=false){
+        let tag = this.chat.taggednicks.get(username.toLowerCase());
+        tag = tag ? `msg-tagged msg-tagged-${tag}` : "";
```

Comment:
How about?:
```javascript
const tag = this.chat.taggednicks.get(username.toLowerCase())
    ? `msg-tagged msg-tagged-${tag}`
    : '';
```

Reply:
As far as I see this does not work because although the get() statement is evaluated, the assignment in the `?`-clause still see the `tag` variable as undefined.

Reply:
huh??

Reply:
You meant to replace both original lines, right?
So we would have something like:
```
x = 5 ? x+1 : -1
ReferenceError: x is not defined
```
?

Reply:
oh right 😦